### PR TITLE
Return 410s on certain URLs to shut up Bingbot

### PR DESCRIPTION
**Pull request by alexwlchan**
Another go at #982.

### CODE REVIEW

File: docker/nginx/loris.nginx.conf [hunk 1]
```diff
@@ -31,6 +31,22 @@ http {
...
+    # A while back, the Collection site started serving all content through
+    # Loris, but requesting JPG format.  This breaks when you request a
+    # transparent PNG, because it falls over trying to fill in the transparent
+    # regions, which triggers a 500 alarm in Slack.
+    #
+    # But those URLs don't exist any more, right?
+    #
+    # Nope, Bingbot keeps requesting them.  So this is meant to shut it up,
+    # eventually we'll remove these, but 410s should tell it to go away.
+    location /image/wordpress:2016/12/victoria_and_albert_christmas_tree.png/full/320,/0/default.jpg {
```
Comment (kenoir) [inline docker/nginx/loris.nginx.conf]: Should we be serving pngs at all? can we make this a pattern for all pngs?

Comment (kenoir) [reply docker/nginx/loris.nginx.conf]: Or maybe something like this: https://stackoverflow.com/questions/8237761/is-it-possible-to-change-the-http-status-code-returned-when-proxy-pass-gateway-i

Comment (alexwlchan) [reply docker/nginx/loris.nginx.conf]: We do serve PNGs, and most of them would serve fine as JPEGs – if they don’t have transparent regions. This is just a short-term hack to get Bingbot to leave these URLs alone.

Comment (kenoir) [reply docker/nginx/loris.nginx.conf]: Yup. I just don't like adding these one off things to the nginx config, with no exact timescale for removing them.

**Pull request merged**